FundacionParaguaya/MentorApp
# Resolve sending dublicate data

**Describe the changes**
The issue is resolved by submitting the draft during the Final instead of the Overview screen.
SubmitDraft action is moved to the final screen.
The button name is changed to Save & Close.

**To Test**
Specific steps to test when reviewing:
1. Take a survey and get to overview screen, check that the draft is not sent
2. Go to final screen, check that draft is not sent
3. Click save&close button, check that the draft is sent
4. Go back and forth between overview and final screen to make sure draft is not sent there

**Related issues**
- Resolves #246
